doc/userguide/capture-hardware/napatech.rst

```diff
@@ -232,27 +237,163 @@ Next execute those command using the ntpl tool::
 
 	$ /opt/napatech3/bin/ntpl -f <my_ntpl_file>
 
-Now you are ready to start suricata::
+Now you are ready to start Suricata::
 
 	$ suricata -c /usr/local/etc/suricata/suricata.yaml --napatech --runmode workers
 
 It is possible to specify much more elaborate configurations using this option. Simply by
-creating the appropriate NTPL file and attaching suricata to the streams.
+creating the appropriate NTPL file and attaching Suricata to the streams.
+
+Bypassing Flows
+---------------
+
+On flow-aware Napatech products traffic from individual flows can be automatically 
+dropped or, in the case of inline configurations, forwarded by the hardware after 
+an inspection of the initial packet(s) of the flow by Suricata.  This will save 
+CPU cycles since Suricata does not process packets for a flow that has already been 
+ajudicated. This is enabled via the hardware-bypass option in the Napatech section 
+of the configuration file.
 
+When hardware bypass is used it is important that the ports accepting upstream 
+and downstream traffic from the network are configured with information on
+which port the two sides of the connection will arrive. This is needed for the 
+hardware to properly process traffic in both directions.  This is indicated in the 
+"ports" section as a hyphen separated list of port-pairs that will be receiving 
+upstream and downstream traffic E.g.:: 
+
+    napatech:
+      hardware-bypass: true
+      ports[0-1,2-3]
+
+Note that these "port-pairings" are also required for IDS configurations as the hardware
+needs to know on which port(s) two sides of the connection will arrive. 
+
+For configuations relying on optical taps the two sides of the pairing will typically 
+be different ports.  For SPAN port configurations where both upstream and downstream traffic 
```

Comment:
Double space before "For"

Reply:
There are more parts with those double spaces, should be easily be fixed throughout the whole documentation part.